backend/handlers/setup/setuphandler.go

```diff
@@ -0,0 +1,22 @@
+package setup
+
+import (
+	"encoding/json"
+	"net/http"
+	"socialpredict/setup"
+)
+
+func GetSetupHandler(w http.ResponseWriter, r *http.Request) {
+
+	appConfig, err := setup.LoadEconomicsConfig()
+	if err != nil {
+		http.Error(w, "Failed to load economic config", http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	err = json.NewEncoder(w).Encode(appConfig.Economics)
+	if err != nil {
+		http.Error(w, "Failed to encode response", http.StatusInternalServerError)
+	}
+}
```

Comment:
Future work question: should we also think about letting an admin user update the setup via a POST to this same endpoint?

Reply:
Great idea!

Reply:
Put together a draft ticket on this for project v0.0.5 https://github.com/orgs/openpredictionmarkets/projects/5?pane=issue&itemId=76289189